api/model/storage/postgres/dataset.go

```diff
@@ -389,6 +412,10 @@ func (s *Storage) AddField(dataset string, storageName string, varName string, v
 	}
 
 	sql := fmt.Sprintf("ALTER TABLE %s ADD COLUMN \"%s\" %s;", storageName, varName, postgres.MapD3MTypeToPostgresType(varType))
+	if len(defaultVal) > 0 {
+		sql = fmt.Sprintf("ALTER TABLE %s ADD COLUMN \"%s\" %s DEFAULT '%s';", storageName, varName, postgres.MapD3MTypeToPostgresType(varType), defaultVal[0])
```

Comment:
Like above, I suggest not using the ...string as a way to pass optional parameters.